hooks/nix.sh

```
@@ -0,0 +1,11 @@
+# Nix setup hook for setting Zirco library paths for shells and builds that USE zrc
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)